agents/rust/mediator/mediation/src/routes/pickup.rs

```diff
@@ -4,66 +4,81 @@ use std::sync::Arc;
 
 use axum::{extract::State, http::StatusCode, Json};
 use log::info;
-
-use crate::{
-    didcomm_types::{
-        pickup_delivery_message_structs::*, PickupDeliveryReqMsg, PickupMsgEnum, PickupStatusMsg,
```

Comment:
I suppose you could also delete these custom structures such as `PickupDeliveryReqMsg` as part of this PR? Or is it still used somehow?

Reply:
Yes. All these need to be removed. In fact, more, the whole router module (mentioned in another comment) needs to go too. 
I intended to remove it as part of another PR where I will be removing all the cruft from mediation crate.


Currently, I am focusing on only plumbing: making things work by using relevant function from this crate.